### PR TITLE
Side Bar New Design

### PR DESCRIPTION
**Pull request by baralogi**
image

### CODE REVIEW

File: components/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { FiChevronDown, FiMenu, FiBell } from 'react-icons/fi';
+
+function NavBar({ onOpen, ...rest }) {
+  return (
+    <Flex
+      ml={{ base: 0, md: 60 }}
+      px={{ base: 4, md: 4 }}
+      height="20"
+      alignItems="center"
+      bg={useColorModeValue('white', 'gray.900')}
```
Comment (nashirjamali) [inline components/NavBar.jsx:25]: is that for dark mode ?

File: components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      borderBottomWidth="1px"
+      borderBottomColor={useColorModeValue('gray.200', 'gray.700')}
```
Comment (nashirjamali) [inline components/NavBar.jsx:27]: this one too

File: components/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    Admin
+                  </Text>
+                </VStack>
+                <Box display={{ base: 'none', md: 'flex' }}>
+                  <FiChevronDown />
+                </Box>
+              </HStack>
+            </MenuButton>
+            <MenuList
+              bg={useColorModeValue('white', 'gray.900')}
```
Comment (nashirjamali) [inline components/NavBar.jsx:65]: I think you should change gray.900 to color what I defined before,
reference from design

File: layouts/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { Box, Drawer, DrawerContent, useColorModeValue, useDisclosure } from '@chakra-ui/react';
+import NavBar from '../components/NavBar';
+import SideBar from '../components/SideBar';
+
+function Main({ children, router }) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <Box minH="100vh" bg={useColorModeValue('gray.100', 'gray.900')}>
```
Comment (nashirjamali) [inline layouts/Main.jsx:8]: change gray.100, gray.900 to what color i defined before

File: pages/index.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, Heading, Text } from '@chakra-ui/react';
+import Body from '../layouts/Body';
+
+function Index() {
+  return (
+    <Body title="Trending">
+      <Box title="Home">
```
Comment (nashirjamali) [inline pages/index.jsx]: Is this title really different from the one above?

Comment (baralogi) [reply pages/index.jsx]: the correct one is the `props ` for the body component

```
<Body title="Trending">
.....
</Body>
```

to make this 

image

i will delete for title props in BOX

File: components/NavBar.jsx (hunk 1 above)
Comment (baralogi) [reply components/NavBar.jsx:25]: Ah yes, soon for DarkMode features. I will fix it later. while I make an issue for it first #3

**Review by nashirjamali — APPROVED** [review]
LGTM (rada maksa xixixi)

**Pull request merged**